docs/gettingstarted/installation-macos.rst

```diff
@@ -18,7 +18,18 @@ PostgreSQL
 
 Saleor needs PostgreSQL version 9.4 or above to work. Get the macOS installer from the `PostgreSQL download page <https://www.postgresql.org/download/macosx/>`_.
 
-Make sure you keep track of the password you set for the administration account during installation.
+Then, create a new database and user. Make sure you keep track of the password you set for the administration account during installation. 
```

Comment:
> Make sure you keep track of the password you set for the administration account during installation.

What do you mean? To make sure to remember the password to access the database (e.g.: development tools)?

Reply:
Actually, I'm a little confused here too. I install Postgres via homebrew. With homebrew there's no setting up an administration account. Are we certain that this occurs when installing via the installer? We might want to remove the whole "Make sure ..." part.